sat/apiclient/elastic.py

```diff
@@ -0,0 +1,37 @@
+#
```

Comment:
This whole file can probably just be removed, I think it was initially added in my "new-command" template. At the time I was thinking we would use an API Client class to connect to elastic